config/puma.rb

```diff
@@ -0,0 +1,43 @@
+# Puma can serve each request in a thread from an internal thread pool.
+# The `threads` method setting takes two numbers: a minimum and maximum.
+# Any libraries that use thread pools should be configured to match
+# the maximum value specified for Puma. Default is set to 5 threads for minimum
+# and maximum; this matches the default thread size of Active Record.
+#
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
+threads min_threads_count, max_threads_count
+
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments.
+#
+worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
+
+# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
+#
+port ENV.fetch("PORT") { 3000 }
```

Comment:
- [ ] Great work implementing the reqyuired changes from the previous reviewer. However, there's little issue here. Your default port value is 3000, and when your app is opened, that should be what you should have on your address bar. (`http://127.0.0.1:3000/`). But when you wanted to fetch the api endpoint from frontend, you called this endpoint :point_right:  `const response = await fetch('http://127.0.0.1:1800/api/v1/greetings')` instead of this :point_right:  `const response = await fetch('http://127.0.0.1:3000/api/v1/greetings');`.
So with the endpoint you used, nothing shows up when I was clicking the `Click me` button until I changed the url to the later endpoint. Kindly look into this and fix.

### App behavious with your endpoint.
[your-endpoint.webm](https://user-images.githubusercontent.com/26597432/236636949-ca2f1ae1-f5c2-4f0e-b977-bf9e8099354a.webm)

### App behavios with my endpoint.
[my-endpoint.webm](https://user-images.githubusercontent.com/26597432/236636954-228b5066-0bb3-4a04-9ebb-329cc70cd656.webm)